rotkehlchen/tests/unit/uniswap/conftest.py

```diff
@@ -1,44 +0,0 @@
-from unittest.mock import patch
```

Comment:
Is this deleted? It does not seem to be. Seems to be just mde into an empty file